README.md

```diff
@@ -1,6 +1,8 @@
-# Getting Started with Create React App
+# Weather App
```

Comment:
Remember your README is the first thing anybody sees when looking at your repo, so it should have all the information needed to install and use your app as well as an explanation of what it does.

Some things you should include/do:
- replace any boilerplate instructions (such as the section about `npm start` and tweak it to be in your own words),
- what technologies are used,
- installation instructions,
- A description of what the app actually does. Does it work for all cities in all countries? What range does it show?
- What would you do if you had more time?
- your name!

Also useful is [markdown editor](https://jbt.github.io/markdown-editor/) so you can play with what the file will look like without having to do multiple commits to the repo to see the results, have a look through this [guide to markdown](https://www.markdownguide.org/cheat-sheet/) too